lib/krill/show_response.rb

```diff
@@ -0,0 +1,79 @@
+module Krill
+
+  # @api krill
+  require 'delegate'
+  class ShowResponse < SimpleDelegator
+
+    # Return the response that was stored under `var`. When used with 
+    # a key associated with a table response-set, returns a list of table responses
+    # in order of the rows of the table.
+    #
+    # @param var [Symbol/String]  The var used to store data under
+    #               specified in the get or select call of the ShowBlock
+    def get_response var
+      responses[var.to_sym]
+    end
+
+    # Returns data recorded in a specified row of an input table
```

Comment:
For all block comments, put a commented blank line between the description and the parameters -- like in the one above.